code/modules/surgery/organs/tongue.dm

```diff
@@ -485,3 +485,20 @@
 		signer.visible_message(span_notice("[signer] raises [signer.p_their()] eyebrows."))
 	else if(question_found)
 		signer.visible_message(span_notice("[signer] lowers [signer.p_their()] eyebrows."))
+
+/obj/item/organ/tongue/stump
```

Comment:
You should probably move this to a modular file. Some where in the customization module likely

Reply:
Alright, I'll do that quickly!

Reply:
Done that now!